src/components/evaluation/EvaluationTable.vue

```diff
@@ -1,17 +1,18 @@
 <template>
-  <div v-if="evaluations.length > 0">
+  <div v-if="evaluations.length">
     <div
-      class="bg-surface-variant elevation-2 sticky"
+      class="bg-surface-variant elevation-2 py-2 sticky"
       role="search"
     >
-      <div class="align-start d-flex flex-wrap justify-start px-5 pt-3" :class="{'pb-2': readonly}">
+      <div class="align-center d-flex flex-wrap px-5 pt-3" :class="{'pb-2': readonly}">
         <v-text-field
           id="evaluation-search-input"
           v-model="searchFilter"
           aria-label="Filter evaluations table by search terms."
-          class="evaluation-search-input mr-3"
+          class="bg-white mr-3"
```

Comment:
@johncrossman If you use bg-surface here instead of bg-white, it will use a dark background in dark mode.

Reply:
Thanks @lyttam.  I'll amend